stix_shifter_modules/alertflex/configuration/alertflex_config.json

```diff
@@ -0,0 +1,55 @@
+{
+    "connection": {
+        "type": {
+            "default": "alertflex"
+        },
+        "host": {
+            "type": "text"
+        },
+        "port": {
+            "default": 8181,
+            "type": "number"
+        },
+        "help": {
+            "default": "https://github.com/olegzhr/cnode/blob/master/doc/integration.rst",
+            "type": "link"
+        },
+        "limit": {
```

Comment:
we move some fields to a different nested object called `options` such as limit, timeout. have a look at base config.json file and remove those fields in that base file unless you want a different default value: https://github.com/opencybersecurityalliance/stix-shifter/blob/develop/stix_shifter_modules/config.json